src/referentielijsten/api/admin.py

```diff
@@ -82,3 +86,15 @@ def is_geldig(self, obj):
             return True
 
         return False
+
+    @admin.display(description=_("Actions"))
+    def items_link(self, obj):
+        url = (
+            # TODO replace with reverse(..., query=...) once Django is upgraded to 5.2
+            reverse("admin:api_item_changelist")
+            + "?"
+            + urlencode({"tabel__id": obj.id})
```

Comment:
Didn't notice this during the first review, I checked the admin views and right now there's no filter option in the sidebar for id. Let's change this to use `code` instead since that's also unique and a bit cleaner to use than the `id`

![afbeelding](https://github.com/user-attachments/assets/ea227a62-75d1-4ddd-897e-855e9422b2d2)

```suggestion
            + urlencode({"tabel__code": obj.code})
```

Could you also add `tabel__code` to the `ItemAdmin.list_filter`? That way it shows the filters accordingly if you navigate to the changelist via this link:

![afbeelding](https://github.com/user-attachments/assets/1198861c-fd05-4b05-b9dc-6d9f505f4c9a)